controllers/core/openstackcontrolplane_controller.go

```diff
@@ -248,6 +245,16 @@ func (r *OpenStackControlPlaneReconciler) reconcileNormal(ctx context.Context, i
 		return ctrlResult, nil
 	}
 
+	// Reconcile optional services when enabled. If the service is explicitly disabled,
+	// we shouldn't try to reconcile it.
+	if instance.Spec.Cinder.Enabled {
```

Comment:
The same is already done in ReconcileCinder function. If we implement this in this top-level then we probably have get the check of Enabled out from individual functions.

Reply:
Yeah, true. I should add that to this PR as well.